app/pubspec.yaml

```diff
@@ -86,6 +86,8 @@ dependencies:
   docx_template: ^0.4.0
   enum_to_string: ^2.0.1
   visibility_detector: ^0.4.0+2
+  firebase_messaging: ^15.0.4
+  flutter_local_notifications: ^17.2.1+2
```

Comment:
I think we already use `awesome_notifications`, can we keep using only that one?

Reply:
Sure, will work on the fix.